config/default.json

```diff
@@ -14,5 +14,29 @@
       "title": "Adopteitor API",
       "description": "Main adopteitor API. <br> https: //github.com/adopteitor"
     }
+  },
+  "authentication": {
```

Comment:
Is this to prepare for future authentication features? Is it correct to commit this secret into source control?

Reply:
Basic authentication via local or jwt it is done with this PR.

This is default conf, so if we are going to distribute this, it doesnt matter. (obiously we dont have to distribute the production keys).

Reply:
Got it! 👍